docs/pages/inboxes/build-inbox.mdx

```diff
@@ -325,11 +335,12 @@ You can configure an XMTP client with these parameters of `Client.create`:
-| dbEncryptionKey | `REQUIRED`                                      | A 32-byte `ByteArray` used to encrypt the local database.                                                                                                                                                                                                                                                                                                                                                                                                  |
-| historySyncUrl  | `https://message-history.dev.ephemera.network/` | The history sync URL used to specify where history can be synced from other devices on the network. For production apps, use `https://message-history.production.ephemera.network`                                                                                                                                                                                                                                                                         |
-| appVersion      | `undefined`                                     | Add a client app version identifier that's included with API requests.<br/>For example, you can use the following format: `appVersion: APP_NAME + '/' + APP_VERSION`.<br/>Setting this value provides telemetry that shows which apps are using the XMTP client SDK. This information can help XMTP core developers provide support to app developers, especially around communicating important SDK updates, including deprecations and required updates. |
+| env             | `DEV`                                           | Connect to the specified XMTP network environment. Valid values include `DEV`, `PRODUCTION`, or `LOCAL`. For important details about working with these environments, see [XMTP DEV, PRODUCTION, and LOCAL network environments](#xmtp-dev-production-and-local-network-environments). |
+| apiURL             | undefined                                           | Manually specify an API URL to use. If specified, value of `env` will be ignored. |
+| appContext (Android-only)     | null                                      | Required. The app context used to create and access the local database. |
+| dbEncryptionKey | null                                      | Required. A 32-byte `ByteArray` used to encrypt the local database.    |
+| dbDirectory | xmtp_db                                      | Optional. Specify a database directory. If no directory is specified, the value is set to `xmtp_db` by default. |
+| historySyncUrl  | `https://message-history.dev.ephemera.network/` | The history sync URL used to specify where history can be synced from other devices on the network. For production apps, use `https://message-history.production.ephemera.network`. |
```

Comment:
Should we add any of the following options?

keystoreProviders, persistConversations, skipContactPublishing, codecs, maxContentSize, preCreateIdentityCallback, preEnableIdentityCallback, basePersistence, apiClientFactory

Reply:
no